modules/docs/mdx/versionsTable.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import {Table, StatusIndicator, StatusIndicatorVariant} from '@workday/canvas-kit-preview-react';
+// @ts-ignore: Cannot find module error
+import {version} from '../../../lerna.json';
+
+// When we release a new version, add the support version before the first item.
+const allVersions = [
+  {
+    versionNumber: version, // This will always be the current major version
+    documentation: 'https://github.com/Workday/canvas-kit',
+  },
+  {
+    versionNumber: 9, // This is support, update this when we release v11
+    documentation: 'https://d2krrudi3mmzzw.cloudfront.net/v9/?path=/docs/welcome--page',
+  },
+  {
+    versionNumber: 8,
+    documentation: 'https://d2krrudi3mmzzw.cloudfront.net/v8/?path=/docs/welcome--page',
+  },
+  {
+    versionNumber: 7,
+    documentation:
+      'https://d2krrudi3mmzzw.cloudfront.net/v7/?path=/story/welcome-getting-started--page',
+  },
+  {
+    versionNumber: 6,
+    documentation:
+      'https://d2krrudi3mmzzw.cloudfront.net/v7/?path=/story/welcome-getting-started--page',
+  },
+];
+
+const statusIndicators = {
+  stable: {
+    variant: 'blue',
+    label: 'Stable',
+  },
+  support: {
+    variant: 'green',
+    label: 'Support',
+  },
+  deprecated: {
+    variant: 'red',
+    label: 'No Longer Supported',
+  },
+};
+function getVersionStatusIndicator(versionNumber: number | string) {
+  const currentMajorVersion = Number(version.split('.')[0]);
+  // version from lerna is a string, so we need to do modify into a number
```

Comment:
This comment should be above line 47. `version` is the version from `lerna.json`.